### DISCUSSION TITLE
Check col track automatically

### DISCUSSION THREAD

**Discussion by snippet2021** [yasserfarouk/scml#104] [General]
@yasserfarouk 
I emailed you yesterday but have not received a reply so I am posting here.

when I submit my agent through the website, the collusion track is automatically checked, even if I have unchecked it.

(If I press “update” with only “run Std” checked, "run col" will also be checked automatically .)

So the only option is to join the collusion track. Please check the issue.

**Comment by yasserfarouk** [comment]
The checkboxes affect joining the official competition which will start after the submission deadline on June 10th. 

We run all compatible agents in both standard and collusion independent from this setting in the online unofficial competition that is currently running. The reason is to give the developer more feedback as rare exceptions may appear in one track but not the other.

**Reply by snippet2021** [reply]
At this point both checkboxes(std and col) are checked. Will the organizers decide which track to participate in based on my report?

**Reply by yasserfarouk** [reply]
If both are checked then you will participate in standard for sure and in collusion if your report contains any nontrivial collusion strategy as per the CFP.

If you uncheck one of them, then you opt out from participating in that track independent of your report.

**Reply by snippet2021** [reply]
Now, we can't uncheck one of them, but do you mean we can uncheck one of them when we submit our report after 6/10?
